predicators/utils.py

```diff
@@ -1733,14 +1735,17 @@ def abstract(state: State, preds: Collection[Predicate]) -> Set[GroundAtom]:
     atoms = set()
     for pred in preds:
         for choice in get_object_combinations(list(state), pred.types):
-            if pred.holds(state, choice):
+            if pred.holds(state, choice, no_load=no_load):
                 atoms.add(GroundAtom(pred, choice))
     return atoms
 
 
 def abstract_from_last(
-        state: State, preds: Collection[Predicate], last_state: State,
-        last_atoms: Set[GroundAtom]) -> Set[GroundAtom]:  # pragma: no cover
+        state: State,
+        preds: Collection[Predicate],
+        last_state: State,
+        last_atoms: Set[GroundAtom],
+        no_load: bool = False) -> Set[GroundAtom]:  # pragma: no cover
```

Comment:
could we maybe create a command line flag for this? Also, when we save the ground atoms dataset, could we add either `fast_grounding` or `slow_grounding` to the name (ditto when loading)? Just want to ensure that we know which type of grounding was used to make a particular dataset, since that might be relevant later (what are your thoughts?)